change/@internal-react-composites-e27cd3a3-4cde-4372-b24e-aa332aee12b2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Make filesharing responsive: prevent file cards from getting trimmed in case of smaller frame",
```

Comment:
Current implementation isn't actually preventing cards from getting trimmed in a smaller frame, it is simply circumventing the issue for our chat sample when the participant pane is open.
If you run the chat sample in the chrome emulator, with participant pane closed, on a samsung galaxy fold device, are the file cards still getting trimmed?